<?xml version="1.0"?>
<launch>
	<include file="$(find assignment_2_2023)/launch/assignment1.launch"/>
    
    <param name = "des_x" value = "5.3"/>
    <param name = "des_y" value = "2.1"/>
    <param name = "avg_win" value = "5"/>
    <node pkg = "assignment_2_2023" type = "reach_position" name = "go_to_point_client" output = "screen"/>
    <node pkg="assignment_2_2023" type="last_target.py" name="last_target"  />
    <node pkg="assignment_2_2023" type="get_dist_avg_speed.py" name="Dist_AvgSpeed_Service"  />
</launch>
